js/core/modules_registry.js

```diff
@@ -0,0 +1,21 @@
+const registry = {};
+
+export function setModule(name, module) {
+    registry[name] = module;
+}
```

Comment:
I suggest you use name registration function in the same manner we did it for ComponentRegistrator:
```suggestion
export function registerModule(name, module) {
    registry[name] = module;
}
```